rpm/stackdriver-agent

```diff
@@ -109,19 +109,19 @@ gen_config() {
         # See if the instance has the correct monitoring scopes.
         INSTANCE_SCOPES=$(curl --silent --connect-timeout 1 -f -H "Metadata-Flavor: Google" http://169.254.169.254/computeMetadata/v1/instance/service-accounts/default/scopes 2>/dev/null || /bin/true)
         if [ `echo "$INSTANCE_SCOPES" | grep -cE '(monitoring.write|monitoring|cloud-platform)$'` -lt 1 ]; then
-            log_failure_msg "The instance has neither the application default" \
+            echo >&2 "The instance has neither the application default" \
```

Comment:
do we need a space after "default" ?

Reply:
Actually, the space before the `\` is sufficient. Shell... 🤷‍♂️

Reply:
🤦‍♂️